ruby2.5/cookiecutter-aws-sam-hello-ruby/{{cookiecutter.project_name}}/tests/unit/test_handler.rb

```diff
@@ -82,13 +82,13 @@ def expected_result
       statusCode: 200,
       body: {
         message: 'Hello World!',
-        location: '1.1.1.1'
+#         location: '1.1.1.1'
```

Comment:
Why did we commented out this and line 91?

Reply:
Because the `app.rb` has it commented our too, so I keep them consistent with each other

https://github.com/aws/aws-sam-cli-app-templates/blob/4be90d257388c2fe412978e53b21a2673fcdbe5e/ruby2.7/cookiecutter-aws-sam-hello-ruby/%7B%7Bcookiecutter.project_name%7D%7D/hello_world/app.rb#L34-L37